internal/infra/pull/sql_datasource.go

```diff
@@ -110,12 +111,12 @@ func (ds *SQLDataSource) RowReader(source pull.Table, filter pull.Filter) (pull.
 		fmt.Fprint(sql, ds.dialect.Limit(filter.Limit()))
 	}
 
-	if ds.logger != nil {
+	if log.Logger.GetLevel() <= zerolog.DebugLevel {
```

Comment:
Have you check by debugging , if this test really return false if the zerolog logger is in info or upper level ?

Reply:
I'll add a test on this topic, checking if the log is here for an info level (if it's ok, then it should be for the upper levels)

Reply:
see https://github.com/CGI-FR/LINO/pull/18/commits/a53404cf61ec4c264d91ab4266f172bc2c4882b4